azurecompute-arm/src/test/java/org/jclouds/azurecompute/arm/features/VaultApiLiveTest.java

```diff
@@ -0,0 +1,1220 @@
+      String certPem = IMPORTABLE_CERTIFICATE_PEM;
+      CertificateBundle certBundle = api().importCertificate(
+              vaultUri,
+              IMPORTABLE_CERTIFICATE_NAME,
+              null,
+              CertificatePolicy.create(
+                      null,
+                      null,
+                      null,
+                      null,
+                      null,
+                      SecretProperties.create("application/x-pem-file"),
+                      null
+              ),
+              null,
+              null,
+              certPem);
+      assertNotNull(certBundle);
+   }
+
+   @Test(dependsOnMethods = "testImportCertificate")
+   public void testMergeCertificate() {
+      /* XXX - Merging certificates is used in the case where a CSR is generated
+       * within the Azure Key Vault and then signed by an external entituy.
+       * Since this requires an offline process outside the scope of automated
+       * tests, this test is currently not implemented.
+       */
+      assertTrue(true);
+   }
+
+   @Test(dependsOnMethods = "testGetCertificateOperation")
+   public void testGetCertificate() {
+      CertificateBundle certBundle = api().getCertificate(vaultUri, "myCertificate");
+      assertNotNull(certBundle);
+   }
+
+   @Test(dependsOnMethods = "testGetCertificate")
+   public void testListCertificates() {
+      List<Certificate> certs = api().getCertificates(vaultUri);
+      for (Certificate cert : certs) {
+         assertNotNull(cert.id());
+      }
+   }
+
+   @Test(dependsOnMethods = "testUpdateCertificate")
+   public void testListCertificateVersions() {
+      List<Certificate> certs = api().getCertificateVersions(vaultUri, CERTIFICATE_NAME);
+      assertNotNull(certs);
+      assertEquals(certs.size(), 1);
+   }
+
+   @Test(dependsOnMethods = "testGetCertificatePolicy")
+   public void testUpdateCertificate() {
+      Map<String, String> tags = new HashMap<String, String>();
+      tags.put("selfsigned", "true");
+      CertificatePolicy policy = api().getCertificatePolicy(
+              vaultUri,
+              CERTIFICATE_NAME
+      );
+      assertNotNull(policy);
+      CertificateBundle certBundle = api().updateCertificate(
+              vaultUri,
+              CERTIFICATE_NAME,
+              "",
+              null,
+              policy,
+              tags
+      );
+      assertNotNull(certBundle);
+      assertEquals(certBundle.tags().size(), 1);
+   }
+
+   @Test(dependsOnMethods = "testUpdateCertificate")
+   public void testUpdateCertificateVersion() {
+      // create a new version of the certificate
+      /*
+       * XXX -- update using version complains about needing policy (required input), yet
+       * passing in the same policy results in the error:
+       *
+       * Policy cannot be updated with a specific version of a certificate
+       *
+       * Will uncomment/fix once this issue is resolved.
+       *
+       */
+      assertTrue(true);
```

Comment:
Better throw a SkipException to see the reason why this is not executed in the test output. this will also serve as a reminder for us that this needs to be implemented.